communications/GSOD-applications/GSoD-2020-Project-Report.md

```diff
@@ -0,0 +1,152 @@
+The new templates will make improving the book more accessible by helping new contributors write interoperable content from the start.
+I also documented the community's contribution workflows and processes by engaging with different community members, and used those insights to create guidelines in a short chapter in the Community Handbook.
+This chapter explains all the templates, contribution workflows and guidelines to use them in _The Turing Way_.
+This effort will ensure that there is a cohesive narrative when it is written by a diverse group of authors from around the world.
+
+The following GitHub pull requests and issues, and online content were developed to ensure support for the future contributions and maintenance:
+
+| Task | PRs/Issues/online content |
+| -------- | -------- | 
+| Designing templates     | [#1507](https://github.com/alan-turing-institute/the-turing-way/pull/1507), [#1571](https://github.com/alan-turing-institute/the-turing-way/pull/1571), [#1601](https://github.com/alan-turing-institute/the-turing-way/pull/1601)     |
+|Writing a contribution workflow Chapter| [#1646](https://github.com/alan-turing-institute/the-turing-way/pull/1646)|
+|Revising existing issue templates | [#1598](https://github.com/alan-turing-institute/the-turing-way/pull/1598) |
+| **PRs merged into online resources** | [Chapter on Templates and Workflow](https://the-turing-way.netlify.app/community-handbook/contributing.html), [Sets of templates](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/templates) |
+
+### 3. Improving the Overall Accessibility of the Online Book
+
+**Goal**: *Improving the accessibility of the material by ensuring that the online book has responsive layout for users reading it in different devices or using accessibility apps.*
+
+My work on this goal focused on minimising the overall information loss readers experienced when reading the book with screen readers or on mobile devices. 
+Already, Jupyter Book allows the book's text to reflow to fit all screen sizes and zoom settings. However, images, videos, and tables had to be responsive for _The Turing Way_ to be readable and look great on all devices. 
+I created several issues to help facilitate the migration of image formatting from vanilla Markdown to Markedly Structured Text, which automatically handles responsive styling for images. 
+Some of these issues also encourage contributors to write descriptive ALT text for the book's images that did not have one. 
+This ensures that people using screen readers are able to understand the image's intended message. 
+
+Finally, I wrote custom CSS to create a better layout and responsive styling for tables and videos across the book. 
+Most importantly, I documented the use of custom CSS in a chapter in the book's Community Handbook.
+
+The following GitHub pull requests and issues, and online content were developed to enhance book's accessibility:
+
+| Task | PRs/Issues/online content |
```

Comment:
I did know there were going to be more awesome tables when I made the comment above.

I continue to love them 👾 👾 👾